packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -374,4 +357,23 @@ export default class Run extends SfdxCommand {
     const hint = messages.getMessage('apexTestReportFormatHint', [reportArgs]);
     return hint;
   }
+
+  /** Handles special exceptions where we don't want to return early
+   * with the testRunId.*/
+  private shouldExitEarly(): boolean {
```

Comment:
this feels very unit testable. Are all paths covered in run.test.ts?

Reply:
As discussed earlier, all paths are well covered, but not explicitly by testing the private function. Let me know if you still think we should change this!

Reply:
I'll leave it up to you as we don't really have a solid plan in place.  If it were me I'd unit test it :)

Reply:
thanks for the effort, can you be sure to open a ticket to call this out and document what wasn't working for you?